### ISSUE TITLE
Cannot change page width in knitR

### ISSUE THREAD

**Issue by JamesD73** [yihui/knitr#962]
I have a text table within a report that is fairly wide. The ends of the lines are cut and new rows are created. What options are available besides the options(width=) in R, and the out.width in the knitR chunk. Neither of these work. An option to print to landscape would also be handy. Some example code is provided below.

```{r global_options, include=FALSE}
knitr::opts_chunk$set(echo=FALSE, warning=FALSE, message=FALSE, cache=FALSE,eval=TRUE,warning=FALSE,message=FALSE,out.width=10000,out.height=10000,fig.width=20,fig.height=20,tidy=TRUE,comment=NA)
options(width=10000)
```

**Comment by yihui** [comment]
Please see https://github.com/yihui/knitr/blob/master/CONTRIBUTING.md and also read the documentation before trying chunk options: http://yihui.name/knitr/options/

**Comment by JamesD73** [comment]
Hi,

thank you so much for replying to my comment so quickly! Your help is much appreciated.
Would you please be able to point me to the exact code that will allow me to increase the width of the space where I can print a very wide table to the HTML format without wrapping the lines. I am sure it is in the page you sent me, however my untrained eye cannot see it.

Thank you,
James


On 27 Feb 2015, at 12:06 am, Yihui Xie <notifications@github.com<mailto:notifications@github.com>> wrote:


Please see https://github.com/yihui/knitr/blob/master/CONTRIBUTING.md and also read the documentation before trying chunk options: http://yihui.name/knitr/options/

**Comment by JamesD73** [comment]
Hi,

I found i had to edit the style .css file. once this was done it worked fine.

thanks

On 27 Feb 2015, at 12:06 am, Yihui Xie <notifications@github.com<mailto:notifications@github.com>> wrote:


Closed #962<https://github.com/yihui/knitr/issues/962>.
